File: API/Models/ViewModels/UserViewModel.cs
using System.Collections.Generic;
using GradePortalAPI.Dtos;

namespace GradePortalAPI.Models.ViewModels
{
    public class UserViewModel
    {
        public string Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Username { get; set; }
        public string City { get; set; }
        public string Position { get; set; }
        public ICollection<SkillDto> Skills { get; set; }
    }
}

File: UI/src/app/_models/user-view-model.ts
import { SkillViewModel } from './skill-view-model';

export interface UserViewModel {
  userData: UserData;
  skills: SkillViewModel[];
}

export interface UserData {
  num: number;
  id: string;
  firstName: string;
  lastName: string;
  username: string;
  city: string;
  position: string;
}
